qiskit/transpiler/passmanager.py

```diff
@@ -349,21 +349,22 @@ class StagedPassManager(PassManager):
     pass manager you are not able to modify the individual passes and are only able
     to modify stages.
 
-    By default instances of ``StagedPassManager`` define a typical full compilation
+    By default, instances of ``StagedPassManager`` define a typical full compilation
     pipeline from an abstract virtual circuit to one that is optimized and
     capable of running on the specified backend. The default pre-defined stages are:
 
-    #. ``init`` - any initial passes that are run before we start embedding the circuit to the backend
-    #. ``layout`` - This stage runs layout and maps the virtual qubits in the
-       circuit to the physical qubits on a backend
-    #. ``routing`` - This stage runs after a layout has been run and will insert any
-       necessary gates to move the qubit states around until it can be run on
-       backend's coupling map.
-    #. ``translation`` - Perform the basis gate translation, in other words translate the gates
-       in the circuit to the target backend's basis set
-    #. ``optimization`` - The main optimization loop, this will typically run in a loop trying to
-       optimize the circuit until a condition (such as fixed depth) is reached.
-    #. ``scheduling`` - Any hardware aware scheduling passes
+    #. ``init`` - Initial passes to run before embedding the circuit to the backend.
+    #. ``layout`` - Maps the virtual qubits in the circuit to the physical qubits on
+       the backend.
+    #. ``routing`` - Inserts swap gates as needed to move the qubit states around until
+       the circuit can be run with the chosen layout on the backend's coupling map.
```

Comment:
I originally worded this to avoid calling out swaps because the routing stage doesn't necessarily need to insert swaps, I'm thinking of techniques like: https://arxiv.org/pdf/1907.02686.pdf which use other gates to move the states around.

Reply:
Ok, I removed the word "swap"